### ISSUE TITLE
fix some color contrast issues

### ISSUE THREAD

**Issue by susanev** [chef/automate#3817]
## overview
fix some a11y issues across automate related to color contrast. these were found using the [wave chrome extension](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US).

## details
- [ ] event feed summary labels need to be chef-primary-dark instead of chef-dark-gray since they are on a gray background
- [ ] big filter button labels across automate need to be chef-primary-dark instead of chef-dark-gray since they are on a gray background
- [ ] status colors across automate need to be updated to use accessible colors
- [ ] apps icons in right-panel need to use the same color as the text
- [ ] purple color on event feed is not accessible, update to using the purple disconnected color from apps
- [ ] control status when its 0 is not accessible @jonong1972 will inform how to fix
